backend/databasemodel/model.sql

```diff
@@ -1513,14 +1513,14 @@ INSERT INTO kooste.tamperewfs_metadata (
 -- object: kooste.museovirastoarcrest_rkykohteet | type: TABLE --
 -- DROP TABLE IF EXISTS kooste.museovirastoarcrest_rkykohteet CASCADE;
 CREATE TABLE kooste.museovirastoarcrest_rkykohteet (
-	"objectId" bigint NOT NULL,
-	geom geometry(MULTIPOINT, 3067) NOT NULL,
+	"ID" bigint NOT NULL,
+	geom geometry(MULTIPOINT, 4326) NOT NULL,
 	visibility boolean DEFAULT True,
 	name text NOT NULL,
 	tarmo_category text DEFAULT 'Nähtävyydet',
 	type_name text DEFAULT 'Rakennettu kulttuurikohde',
 	www text,
-	CONSTRAINT museovirastoarcrest_rkykohteet_pk PRIMARY KEY ("objectId")
+	CONSTRAINT museovirastoarcrest_rkykohteet_pk PRIMARY KEY ("ID")
```

Comment:
Same here; Just replace objectId with OBJECTID, ID is not unique field (see my comment in upgrade.sql)